schemas/external/repo/linked-asset-relationship.descriptor.json

```diff
@@ -0,0 +1,7 @@
+{
+  "@id": "https://ns.adobe.com/descriptors/sample-asset-relationship-descriptor",
```

Comment:
That is not a valid JSON-LD ID.  You probably want this to be `xdm:id` instead.

Reply:
Descriptors use an @id and @type - https://github.com/adobe/xdm/blob/master/schemas/common/descriptors/relationshipdescriptor.example.1.json

What makes this URI not valid?